dev-python/node-semver/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```

Comment:
@gentoo/python, the ebuild looks trivial. Do we want to take the package as a dep? Or as backup maintainer if @comio is willing to maintain it?